Gemfile.lock

```diff
@@ -0,0 +1,177 @@
+PATH
```

Comment:
This should be removed, the `.gemspec` should be all the loose Ruby dependencies we need.